### ISSUE TITLE
Config file should also read config in ~ en /etc if no config found in current directory

### ISSUE THREAD

**Issue by ruurd** [ezekg/git-hound#11]
Config file should also read config in ~ and /etc if no config found in current directory
As an alternative I would suggest reading the default config from a setting in ~/.gitconfig, for example:

```
[hound]
    config = /etc/hound/default.yml
```

**Comment by ezekg** [comment]
You can provide an alternative path for the config using the `-config` flag, e.g.:

```bash
git hound -config=`realpath ~/.githound.yml` sniff
```

But I do like the idea of using `.gitconfig` to configure a default if desired. I'd be open to a PR. 🙂

**Comment by ruurd** [comment]
OK that's crude and invokes a subshell. I was wondering why ~/.githound.yml was not working. I would expect that to be shellexpanded but no - git-hound maintains that the config file is not found.

**Comment by ezekg** [comment]
@ruurd yeah, the `~` is not currently auto-expanded. Open to a PR to fix that, though.

**Comment by ezekg** [comment]
Might be a good dep to pull in: https://github.com/mitchellh/go-homedir

Handles a lot of edge cases and platform-dependent issues.
